gravitee-apim-console-webui/src/management/integrations/integration-overview/integration-overview.component.html

```diff
@@ -0,0 +1,112 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<div class="integrations-overview">
+  <div class="page-header">
+    <h1 class="page-header__page-title">Overview</h1>
+    <span class="page-header__description body-strong">Monitor and manage your integration with the provider</span>
+  </div>
+
+  <mat-card>
+    <div class="card-header">
+      <div class="card-header__title">
+        <h3>{{ integration?.name }}</h3>
+        <p *ngIf="integration?.description.length" class="description">{{ integration?.description }}</p>
+      </div>
+    </div>
+
+    <div class="card-content">
+      <mat-card>
+        <div class="details-card">
+          <div class="details-card__item">
+            <h5 class="title body-strong">Provider</h5>
+            <div class="name">
+              <img class="logo" [src]="'../../../assets/logos/' + integration?.provider + '.svg'" alt="Integration provider logo" />
+              <span class="body-strong">{{ integration?.provider }}</span>
+            </div>
+          </div>
+
+          <div class="separator"></div>
+
+          <div class="details-card__item">
+            <h5 class="title body-strong">Agent Connection</h5>
+            <div class="status">
+              <span *ngIf="integration?.agentStatus === 'DISCONNECTED'; else connected" class="gio-badge-error">
+                <mat-icon class="icon-left" svgIcon="gio:prohibition"></mat-icon>
+                Disconnected
+              </span>
+
+              <ng-template #connected>
+                <span class="gio-badge-success">
+                  <mat-icon class="icon-left" svgIcon="gio:check"></mat-icon>
+                  Connected
+                </span>
+              </ng-template>
+            </div>
+          </div>
+          <div class="separator"></div>
+          <div class="details-card__item">
+            <h5 class="title body-strong">Integration ID</h5>
+            <div class="status">
+              <span class="gio-badge-neutral">
+                {{ integration?.id }}
+                <mat-icon
+                  matTooltip="Copy to clipboard"
+                  (click)="copyToClipboard()"
+                  class="icon-right copy-icon"
+                  svgIcon="gio:copy"
+                ></mat-icon>
+              </span>
```

Comment:
We have a component that handle that copy feature: https://particles.gravitee.io/?path=/story/components-clipboard--inner-button-always-visible should we use it?

Reply:
absolutely 👍,  looks slightly different than in design, however we should use particles first.